src/FSharpPlus/Control/Collection.fs

```diff
@@ -69,6 +70,7 @@ type OfList =
     static member        OfList ((x: list<KeyValuePair<'k,'v>>, _: IDictionary<'k,'v>              ), _: Default4) = x |> List.map (|KeyValue|) |> dict
     static member        OfList ((x: list<'k*'v>              , _: IDictionary                     ), _: Default4) = let d = Hashtable () in x |> List.iter d.Add; d :> IDictionary
     static member        OfList ((x: list<KeyValuePair<'k,'v>>, _: IDictionary                     ), _: Default4) = let d = Hashtable () in x |> List.iter (function (KeyValue x) -> d.Add x); d :> IDictionary
+    static member        OfList ((x: list<'t>                 , _: IReadOnlyCollection<'t>         ), _: Default4) = IReadOnlyCollection.ofSeq x
```

Comment:
It would be nice to add a line around here to test this conversion:
https://github.com/fsprojects/FSharpPlus/blob/61e0e1cd3688fd94a742077c3cdb5a01d4f2d7bb/tests/FSharpPlus.Tests/General.fs#L612-L613

Reply:
Done